externals/paho-mqtt/v0/src/index.kt

```diff
@@ -0,0 +1,63 @@
+//headers for Paho JavaScript Client
+//The Paho JavaScript Client is an MQTT browser-based client library written in Javascript that uses WebSockets to connect to an MQTT Broker.
+//https://github.com/eclipse/paho.mqtt.javascript
+
+package js.externals.paho_mqtt
+
+external object Paho {
+    object MQTT {
+        class Client(hostname: String, port: Int, clientId: String) {
+            val hostname: String
+            val port: Int
+            val clientId: String
+            fun connect(options: Options)
+            fun send(message: Message)
+            fun subscribe(destination: String)
+            fun disconnect()
+            var onMessageArrived: (msg: Message) -> Unit
+            var onConnectionLost: (response: Response) -> Unit
+        }
+
+        class Message(payloadString: String) {
+            val payloadString: String
+            var destinationName: String
+        }
+
+        class Response {
+            val errorCode: Int
+            val errorMessage: String
+        }
+
+        interface Options {
```

Comment:
IMHO `ConnectOptions` is a better name for this, also looks like most of properties can be optional